app/helpers/application_helper.rb

```diff
@@ -1,2 +1,14 @@
 module ApplicationHelper
+  def rating_options(selected=nil)
+    options_for_select([["1 star", 1.0],["2 stars", 2.0],["3 stars", 3.0],["4 stars", 4.0],["5 stars", 5.0]], selected)
+  end
+
+  def video_rating_by_user(video_id,user_id)
+    review = Review.find_by_video_id_and_user_id(video_id,user_id)
+    if review
+      return review.rating
+    else
+      return nil
+    end
+  end
```

Comment:
This method is wrong form many points, it shows that you don't fully understand `return` keyword and explicit / implicit return. Revisit this topic as its important to know this. Second point is that this kind of logic is not a helper and presentation logic. You are retrieving data from database and its that kind of model-business logic so you should aim to have this method in one of the models. Also you are passing in ID's to fetch a record.

Definitely this method needs a rework.